opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -226,6 +227,26 @@ def is_recording_events(self) -> bool:
         events with the add_event operation and attributes using set_attribute.
         """
 
+    def __enter__(self) -> "Span":
+        """Invoked when `Span` is used as a context manager.
+
+        Returns the `Span` itself.
+        """
+        return self
+
+    def __exit__(
+        self,
+        exc_type: typing.Optional[typing.Type[BaseException]],
+        exc_val: typing.Optional[BaseException],
+        exc_tb: typing.Optional[python_types.TracebackType],
+    ) -> bool:
```

Comment:
```suggestion
    ) -> Optional[bool]:
```

According to https://github.com/python/mypy/blob/master/docs/source/protocols.rst#context-manager-protocols, and IIRC most context managers like this return null.

Reply:
Yeah, I was wondering about this one later on (specially about the `null` case - I'm wondering why there's nothing for this already available in the `typing` module ;) )